src/DIRAC/FrameworkSystem/Agent/ComponentSupervisionAgent.py

```diff
@@ -565,8 +569,8 @@ def _checkServiceURL(self, serviceName, options):
     module = options['Module']
     self.log.info('Checking URLs for %s/%s' % (system, module))
     urlsConfigPath = PathFinder.getSystemURLSection(
-        serviceName=os.path.join(system, module),
-        setup=self.setup
+        system=system,
```

Comment:
Without the Module, this just cannot find the correct URLs

Reply:
This will get the section, let's say it returns /Systems/WorkloadManagement/Certification/URLs. The URLs are actually taken 2 lines below.

Reply:
But it should return
/Systems/WorkloadManagement/Certification/URLs/JobMonitor

Reply:
I think my bad, this was always the wrong thing to use.